tests/unit/drivers/test_querylang_drivers.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+from jina.executors.crafters import BaseSegmenter
+from jina.flow import Flow
+from jina.proto import jina_pb2
+from tests import JinaTestCase
+
+
+def random_docs(num_docs):
+    for j in range(num_docs):
+        d = jina_pb2.Document()
+        d.id = j
+        d.text = 'hello world'
+        for m in range(10):
+            dm = d.matches.add()
+            dm.text = 'match to hello wolrd'
+            dm.id = m
+            dm.score.ref_id = d.id
+        yield d
+
+
+class DummySegmenter(BaseSegmenter):
+
+    def craft(self, text, *args, **kwargs):
+        return [{'text': 'adasd' * j} for j in range(10)]
+
+
+class MyTestCase(JinaTestCase):
```

Comment:
Could we rename these `MyTestCase` to something more descriptive? Like QueryLangTestCase? I am trying to do it slowly when I touch some part related to them